test/modules/core/shaderlib/project/project-glsl.spec.js

```diff
@@ -195,8 +190,8 @@ const TEST_CASES = [
         name: 'project_position',
         func: ({project_position}) => project_position([-122.05, 37.92, 0, 1], [0, 0]),
         output: getPixelOffset(
```

Comment:
Is it worth adding some comments to this file? I haven't read it too carefully but I am not sure I would be able to easy understand at a glance where high precision tests are being made. Maybe it shouldn't matter?